libseccomp/src/lib.rs

```diff
@@ -41,25 +41,34 @@
 //! ```
 
 pub mod error;
+#[cfg(libseccomp_v2_5)]
 pub mod notify;
 
 use error::ErrorKind::*;
 use error::{Result, SeccompError};
 use libseccomp_sys::*;
 use std::convert::TryInto;
 use std::ffi::{CStr, CString};
+use std::fmt;
 use std::os::unix::io::AsRawFd;
 use std::ptr::NonNull;
 
-/// ScmpVersion represents the version information of
-/// the currently loaded libseccomp library
+/// ScmpVersion represents the version information of the libseccomp library
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
 pub struct ScmpVersion {
     pub major: u32,
     pub minor: u32,
     pub micro: u32,
 }
 impl ScmpVersion {
+    pub fn new(major: u32, minor: u32, micro: u32) -> Self {
```

Comment:
Is it better to create a macro instead of the `new` method?
I think it is little complicated for some users to distinguish the `new` and the `current` method by the name.

Reply:
What about making it private? `ScmpVersion { major: 3, minor: 2, micro: 0 }` works too.

Or `fn from(version: (u32, u32, u32)) -> Self`?

Reply:
This `new` method will be helpful for users who call `check_api` and `check_version` that are `public` function, so it is better to make it `public`. I think writing `ScmVersion { major: 3, minor: 2, micro: 0 } ` is little tiring, so if possible, I want to use the mehod like `new`.
`from` sounds good! I'll change it from the `new` to `from` if you are ok.